pyglet/gl/lib.py

```diff
@@ -102,14 +102,22 @@ def errcheck(result, func, arguments):
         raise GLException('No GL context; create a Window first')
     error = gl.glGetError()
     if error:
-        msg = ctypes.cast(gl.gluErrorString(error), ctypes.c_char_p).value
+        # These are the 6 possible error codes we can get in opengl core 3.3+
```

Comment:
This is much nicer, thanks!

Reply:
Not really any choice when we don't have `gluErrorString` 😄